ocs_ci/ocs/ui/base_ui.py

```diff
@@ -444,7 +447,7 @@ def navigate_odf_overview_page(self):
             self.ocp_version_full >= version.VERSION_4_10
             and ocs_version >= version.VERSION_4_10
         ):
-            self.do_click(locator=self.page_nav["odf_tab_new"], timeout=90)
+            self.do_click(locator=self.page_nav["odf_tab"], timeout=90)
```

Comment:
Why did you change this locator?

Reply:
Because that's the tag that points to ODF page 
https://github.com/shivamdurgbuns/ocs-ci/blob/bz1999767-new/ocs_ci/ocs/ui/views.py#L461
https://github.com/shivamdurgbuns/ocs-ci/blob/bz1999767-new/ocs_ci/ocs/ui/views.py#L475